hedgehog/src/Hedgehog/Internal/Exception.hs

```diff
@@ -5,7 +5,7 @@ module Hedgehog.Internal.Exception (
   ) where
 
 import           Control.Exception (Exception(..), AsyncException, SomeException(..), evaluate)
-import           Control.Monad.Catch (MonadCatch(..), throwM)
+import           Control.Exception.Safe (MonadCatch(..), catch, throwM)
```

Comment:
```
Error:     The import item ‘MonadCatch(..)’ suggests that
    ‘MonadCatch’ has (in-scope) constructors or class methods,
    but it has none
  |
8 | import           Control.Exception.Safe (MonadCatch(..), catch, throwM)
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Yea, I've seen the CI failures, but I wanted to get feedback as to whether this would be accepted before continuing.